DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py

```diff
@@ -56,6 +56,15 @@ def filterReplicas( opFile, logger = None, dataManager = None ):
     return S_ERROR( failed )
 
   replicas = replicas["Successful"].get( opFile.LFN, {} )
+  if not replicas:
```

Comment:
Right, I should fix that... It was only in case there is no replicas, even not active...